manifests/a/aome510/spotify-player/0.17.1/aome510.spotify-player.installer.yaml

```
@@ -0,0 +1,15 @@
+PackageIdentifier: aome510.spotify-player
+PackageVersion: 0.17.1
+InstallerLocale: en-US
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: spotify_player.exe
+  PortableCommandAlias: spotify_player
+ReleaseDate: 2024-03-13
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 876.)